custom_components/plugwise/strings.json

```diff
@@ -13,6 +13,13 @@
     "flow_title": "Smile: {name}",
     "step": {
       "user": {
+        "title": "Plugwise type",
+        "description": "Product:",
+        "data": {
+          "flow_type": "Connection type"
```

Comment:
This bit might be where the 'flow_action' or thing from #117 came from, but it didn't reproduce for me in Core

Reply:
I've tested with this code present, adding an Adam, both via discovery and manual adding, works as it did before the change.